docs/contributing.md

```diff
@@ -6,17 +6,16 @@ You are welcome to work on any change you’d like to see in `nim-libp2p`.
 
 Feel free to browse the existing issues in the repository, especially those marked as [good first issue](https://github.com/vacp2p/nim-libp2p/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22), which are particularly suitable for newcomers looking to contribute. These issues are designed to be easy and straightforward.
 
-When choosing an existing issue to work on, please ping the core maintainers so they can assign it to you.
+When choosing an existing issue to work on, please ping the core maintainers so they can assign it to you. This will prevent working on same issue and core maintainers may give you some hints. 
 
 ## Perform code reviews
 
 Feel free to let us know if you found anything that can a) speed up the project development b) ensure better quality and c) reduce possible future bugs.
 
 ## Contribute
 
-- **Developing**.Read the [development](docs/development.md) guide to get started with the project and testing the code. Additionally, check the [compile time flags](docs/compile_time_flags.md) page, where all available compile-time flags are documented.
+- **Developing**. Read the [development](https://github.com/vacp2p/nim-libp2p/blob/master/docs/development.md) guide to get started with the project and testing the code. Additionally, check the [compile time flags](https://github.com/vacp2p/nim-libp2p/blob/master/docs/compile_time_flags.md) page, where all available compile-time flags are documented. Reading [common hurdles](https://github.com/vacp2p/nim-libp2p/blob/master/docs/common_hurdles.md) for frequently encountered issues can be helpfull to prevent painfull time loss in advance.
```

Comment:
going with absolute links on this page, becasue this page can be accessed using two links:
- page (when accessed by clicking on contributing link-> https://github.com/vacp2p/nim-libp2p/blob/master/docs/contributing.md
- nav tab (when accessed from github navigation tab) -> https://github.com/vacp2p/nim-libp2p?tab=contributing-ov-file

supporting both links with same relative paths could not be achieved (shame on github). using 
`docs/page.md` or `./docs/page.md` will work for nav tab links, and `page.md` works on page links.

the only way to make it work with both is to use absolute path to docs on master - does not seem like terrible compromise.